Source/Core/Common/FileUtil.cpp

```diff
@@ -913,7 +913,13 @@ std::string GetSysDirectory()
 #endif
   sysDir += DIR_SEP;
 
-  INFO_LOG_FMT(COMMON, "GetSysDirectory: Setting to {}:", sysDir);
+  INFO_LOG_FMT(COMMON, "CreateSysDirectoryPath: Setting to {}", sysDir);
+  return sysDir;
+}
+
+std::string GetSysDirectory()
+{
+  static std::string sysDir = CreateSysDirectoryPath();
```

Comment:
Is there a reason not to make it `static const std::string` and not returning a `const std::string&`?